specification/provisioningservices/resource-manager/Microsoft.Devices/2017-08-21-preview/iotdps.json

```diff
@@ -0,0 +1,1370 @@
+                "type": "string"
+              },
+              {
+                "$ref": "#/parameters/api-version"
+              }
+            ],
+            "responses": {
+              "200": {
+                "description": "This is a synchronous operation. The body contains a JSON-serialized array of shared access policies, including keys, that you can use to access the provisioning service endpoints.",
+                "schema": {
+                  "$ref": "#/definitions/SharedAccessSignatureAuthorizationRuleListResult"
+                }
+              },
+              "default": {
+                "description": "DefaultErrorResponse",
+                "schema": {
+                  "$ref": "#/definitions/ErrorDetails"
+                }
+              }
+            },
+            "x-ms-pageable": {
+              "nextLinkName": "nextLink"
+            }
+          }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/provisioningServices/{provisioningServiceName}/keys/{keyName}/listkeys": {
+          "post": {
+            "tags": [
+              "POST"
+            ],
+            "operationId": "IotDpsResource_GetKeysForKeyName",
+            "summary": "Get a shared access policy by name from a provisioning service.",
+            "description": "Get a shared access policy by name from a provisioning service.",
+            "parameters": [
+              {
+                "name": "provisioningServiceName",
+                "in": "path",
+                "required": true,
+                "type": "string"
+              },
+              {
+                "name": "keyName",
+                "in": "path",
+                "required": true,
+                "type": "string"
+              },
+              {
+                "$ref": "#/parameters/subscriptionId"
+              },
+              {
+                "name": "resourceGroupName",
+                "description": "The name of the resource group that contains the provisioning service.",
+                "in": "path",
+                "required": true,
+                "type": "string"
+              },
+              {
+                "$ref": "#/parameters/api-version"
+              }
+            ],
+            "responses": {
+              "200": {
+                "description": "This is a synchronous operation. The body contains a JSON-serialized shared access policy, including keys, that you can use to access one or more provisioning service endpoints.",
+                "schema": {
+                  "$ref": "#/definitions/SharedAccessSignatureAuthorizationRule"
+                }
+              },
+              "default": {
+                "description": "DefaultErrorResponse",
+                "schema": {
+                  "$ref": "#/definitions/ErrorDetails"
+                }
+              }
+            }
+          }
+        }
+      },
+      "definitions": {
```

Comment:
This is not even valid Swagger/OpenAPI.

You've got `definitions` nested in something - it has to be at the root of the document.